scripts/asr_language_modelling/ngram_lm/eval_beamsearch_ngram.py

```diff
@@ -0,0 +1,186 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+TOKEN_OFFSET = 100
```

Comment:
Lets rename this script to be "ngram_lm_subword/eval_beam_search_ngram_subword.py" - this technique works primarily for subword models so we dont want to confuse users with the char based model LM eval in the notebook.

Reply:
I plan to add support for char based LM too. It would have the exact same input arguments and most of the code is the same. Let's keep the name. I would mention that it supports just BPE for now.